### ISSUE TITLE
Options for the pipeline with all the steps

### ISSUE THREAD

**Issue by agroppi** [Gabaldonlab/redundans#4]
Hi, 
I will use your pipeline on a draft assembly of big genome (500 Mb) from pacbio reads with canu 1.4.
Some questions before : 
in the examples there is : 
```
# reduction, reference-based scaffolding and gap closing with paired-end reads
./redundans.py -v -i test/600_?.fq.gz -r test/ref.fa -f test/contigs.fa -o test/run_ref_pe-closing --noscaffolding
```
In this case for the pipeline with all the steps (reduction, scaffolding and gap-filling), the `--noscaffolding` option shoudn't be there ? Am I right ?
In this example, test/600_?.fq.gz is the illumina reads  ?

Thanks

**Comment by lpryszcz** [comment]
Good point, the descriptions was not precise enough. I've clarified it.
